### PR TITLE
Add a file to be used with poudriere buik -f listing the packages

### PR DESCRIPTION
**Pull request by gvnn3**

### CODE REVIEW

File: morello-pkglist [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Morello architecture on CheriBSD.  It is broken down into two sections, 
+# a set of packages known to build and the set of requeested packages
+# that do not build as yet.
+#
+# This file is meant to be used with the -f argument to poudriere bulk
+#
+# The current list of requested packages is maintained on 
+# https://github.com/CTSRD-CHERI/poudriere-infrastructure/wiki/Package-backlog
+#
+# Currently Working Packages 17 March 2022 <- Keep this current!
```
Comment (kwitaszczyk) [inline morello-pkglist:12]: I'd add an empty line to above this line to separate it from the main comment and make it better visible.

File: morello-pkglist [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+lang/perl5.32
+ports-mgmt/pkg
+security/sudo
+shells/bash
+sysutils/seatd
+x11-toolkits/wlroots
+x11-wm/sway
+
+# Non Working Packages
+
```
Comment (kwitaszczyk) [inline morello-pkglist:26]: I'd remove one empty line.

Comment (kwitaszczyk) [reply morello-pkglist:26]: Also, I'd add an additional comment in a separate line:
> The following ports should be moved to the working list once proved as working.

**Comment by kwitaszczyk** [comment]
Can we move `morello-pkglist` to a separate directory (e.g., pkglist) as we're expecting more package lists to appear? Can we also rename it to `aarch64c-pkglist` to match an architecture name passed to poudriere commands?

I'd expect three lists in the near future: `common-pkglist`, `aarch64c-pkglist` and `riscv64c-pkglist`. `common-pkglist` should contain packages that are expected to work on all CHERI-enabled architectures. For now, `aarch64c-pkglist` would be sufficient.

**Review by kwitaszczyk — CHANGES_REQUESTED** [review]

**Comment by gvnn3** [comment]
A subdir (pkglists ?) seems fine.

**Comment by kwitaszczyk** [comment]
Can we rename `morello` to `aarch64c`?

**Review by kwitaszczyk — APPROVED** [review]
Thanks!

**Pull request merged**